Triplehelix490/experimental-light/experimental-light.ino

```diff
@@ -0,0 +1,47 @@
+// the elements which create the charicteristics of the code \/\/\/\/\/\/
+int pin = 9;
+int brightness = 0;
+int fadeAmount = 5;
+int time = 100;
+int flash = 3;
+
+// the setup pre-program \/
+void setup() {
+  // which pin will be the output \/
+  pinMode(pin, OUTPUT);
+  	Serial.begin(9600);
+}
+// the loop \/
+void loop() {
+  // the brightness of the output component aswell the type of signal sent \/
+  analogWrite(pin, brightness);
+  // change the brightness for the next loop \/
+  brightness = brightness + fadeAmount;
+  // this reverses the direction of the fading at the end of each fading phase\/
+  		if (brightness <= 0 || brightness >= 255) {
+   		 fadeAmount = -fadeAmount;
+ 		 }
+  // this adds the blinking effect into the slow fade \/
+ 			if( divisibleByFive(brightness) == true) {
```

Comment:
Maybe you could put elif?? start with for brightness in fadeAmount after you put if and elif...anyway something like that..